### ISSUE TITLE
wierd textbox on firefox

### ISSUE THREAD

**Issue by fwipsity** [zeondev/lunar#24]
**Describe the bug**
the message textbox is streching a bit to much if there are less than 5 rows.

**To Reproduce**
1. Open lunar on firefox
2. enter a name
3. click the done button
4. See the wierd textbox at the bottom

**Expected behavior**
it should be 1 line shorter until there are 5 rows

**Screenshots**
![bug](https://user-images.githubusercontent.com/79366074/117529951-49787480-afe3-11eb-9b11-a7613e1ed11d.PNG)

**Desktop (please complete the following information):**
 - OS: windows
 - Browser Firefox
 - Version latest

**Comment by datkat21** [comment]
Same issue here on Firefox 88.0.1
![C0vQae0STi](https://user-images.githubusercontent.com/49850194/117542074-af153280-afdc-11eb-81ca-0b87f4ca55d0.gif)
Seems to be partially fixed by removing the `height: auto;` preset from the style
![image](https://user-images.githubusercontent.com/49850194/117542102-d79d2c80-afdc-11eb-8987-46040bbfeafb.png)
but then it doesn't auto size to the text in the textbox.

**Comment by fwipsity** [comment]
yes it will work but if you will try to create a new line the textarea wont expand

**Comment by datkat21** [comment]
That's what I just said:
> but then it doesn't auto size to the text in the textbox.
